frontend/i18n/pl/projects/dataset.js

```diff
@@ -0,0 +1,28 @@
+export default {
+  dataset: 'Zbiór danych',
+  actions: 'Akcje',
+  importDataset: 'Importuj zbiór danych',
+  exportDataset: 'Eksportuj zbiór danych',
+  type: 'Typ',
+  title: 'Tytuł',
+  view: 'Widok',
+  order: 'Kolejność',
+  publishDatetime: 'Opublikowano w',
+  text: 'Tekst',
+  metadata: "Metadane",
+  action: 'Akcja',
+  startAnnotation: 'Rozpocznij adnotację',
+  annotate: 'Opisz',
+  importDataTitle: 'Importuj zbiór danych',
+  importDataMessage1: 'Wybierz format pliku',
+  importDataMessage2: 'Wybierz plik(i)',
+  importDataPlaceholder: 'Plik wejściowy',
+  exportDataTitle: 'Eksportuj zbiór danych',
```

Comment:
I think instead of `Plik wejściowy`, the translation `Wprowadzanie pliku` like the one in frontend/i18n/pl/projects/labels.js is better.